.github/workflows/coverage.yml

```diff
@@ -26,7 +25,15 @@ jobs:
       - name: Run cargo-llvm-cov
         uses: taiki-e/install-action@cargo-llvm-cov
       - name: Generate code coverage
-        run: cargo llvm-cov --workspace --lcov --output-path lcov.info
+        run: |
+          cargo llvm-cov --workspace --no-report --features slow_tests
+          cargo llvm-cov --no-report run --bin=forest-cli -- --chain calibnet snapshot fetch -s .
+          cargo llvm-cov --no-report run --bin=forest -- --chain calibnet --encrypt-keystore false --import-snapshot *.car --detach
+          cargo llvm-cov --no-report run --bin=forest-cli -- sync wait
+          cargo llvm-cov --no-report run --bin=forest-cli -- snapshot export
+          killall --signal SIGINT forest
+          sleep 5
```

Comment:
it's a bit optimistic; maybe let's introduce a bit more determinism here https://unix.stackexchange.com/a/423473/375112

Reply:
Oooh, nice!